Patches/O21 Outland - Redburn Pact/Redburn_WeaponsMelee.xml

```diff
@@ -0,0 +1,478 @@
+								<chanceFactor>0.60</chanceFactor>
+								<armorPenetrationBlunt>0.936</armorPenetrationBlunt>
+								<armorPenetrationSharp>1.88</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>edge</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>31</power>
+								<cooldownTime>1.28</cooldownTime>
+								<chanceFactor>0.30</chanceFactor>
+								<armorPenetrationBlunt>2.6</armorPenetrationBlunt>
+								<armorPenetrationSharp>0.65</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+							</li>
+						</tools>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Outland_PactSword"]/statBases</xpath>
+					<value>
+						<Bulk>7.85</Bulk>
+						<MeleeCounterParryBonus>1.38</MeleeCounterParryBonus>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Outland_PactSword"]</xpath>
+					<value>
+						<equippedStatOffsets>
+							<MeleeCritChance>0.85</MeleeCritChance>
+							<MeleeParryChance>0.69</MeleeParryChance>
+							<MeleeDodgeChance>0.37</MeleeDodgeChance>
+						</equippedStatOffsets>
+					</value>
+				</li>
+
+				<!-- Obsidian Sword -->
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Outland_ObsidianSword"]/statBases</xpath>
+					<value>
+						<Bulk>13</Bulk>
+						<MeleeCounterParryBonus>1.5</MeleeCounterParryBonus>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="Outland_ObsidianSword"]</xpath>
+					<value>
+						<equippedStatOffsets>
+							<MeleeCritChance>0.3</MeleeCritChance>
+							<MeleeParryChance>1.6</MeleeParryChance>
+							<MeleeDodgeChance>0.56</MeleeDodgeChance>
+						</equippedStatOffsets>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="Outland_ObsidianSword"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>edge</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>48</power>
+								<cooldownTime>3.4</cooldownTime>
+								<chanceFactor>1.165</chanceFactor>
+								<armorPenetrationBlunt>10.4</armorPenetrationBlunt>
+								<armorPenetrationSharp>1.8</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>handle</label>
+								<capacities>
+									<li>Poke</li>
+								</capacities>
+								<power>3</power>
+								<cooldownTime>1.59</cooldownTime>
+								<chanceFactor>0.10</chanceFactor>
+								<armorPenetrationBlunt>0.625</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+							</li>
+						</tools>
+					</value>
+				</li>
+
+				<!-- Naginata -->
```

Comment:
I can't find a Naginata in the Redburn Pact mod. Maybe this is just a leftover and can be deleted?

Reply:
Ah yeah I must have pasted twice, was using it as a base to work from, Naginata is in Eastborn Empire so I probably had both loaded when testing it as I didn't spot the error.

Reply:
That's sort of what I figured was the case--it appeared as a duplicate xml error initially and confused me at first. Are you able to make a new commit to remove those couple of operations? It'd probably be quicker than me opening my own PR against your branch.

Reply:
Yep that should be sorted now.